locale/pl/LC_MESSAGES/disco.po

```diff
@@ -15,91 +15,93 @@ msgstr ""
 
 #: src/disco/components/Addon.js:112
 msgid "Close"
-msgstr ""
+msgstr "Zamknij"
 
 #: src/disco/components/Addon.js:141
 msgid "Hover to preview or click to install %(name)s"
 msgstr ""
+"Najechanie wyświetli podgląd, kliknięcie zainstaluje dodatek „%(name)s”"
```

Comment:
![Screenshot](https://cloud.githubusercontent.com/assets/139229/17384180/7f6ee5ec-59da-11e6-9ff9-4ad0e3609f28.png)
Probably. I'm gonna change it.